GammaGeeks/tura-backend
# initialising project and updating readme

### What does this PR do?
- It initializes a NestJS project with Prisma for backend development and database management.

### Description of the task to be completed?
- Creating and structuring a new NestJS project.
- Installing and configuring Prisma.
- Defining the database schema in `schema.prisma`.
- Running initial migrations.
- Generating the Prisma client for use in NestJS services.

### How should this be manually tested?
- Clone the repository and navigate to the project directory.
- Run `npm install` to install dependencies.
- Check that the Prisma client is successfully generated and can be used in services.
- Run `npx prisma migrate dev` to apply migrations and verify the database setup.

### What are the relevant pivotal tracker stories?
[#188511332](https://www.pivotaltracker.com/story/show/188511332)
